packages/app/src/cli/services/bulk-operations/execute-bulk-operation.ts

```diff
@@ -172,3 +186,19 @@ function isMutation(graphqlOperation: string): boolean {
   const operation = document.definitions.find((def) => def.kind === 'OperationDefinition')
   return operation?.kind === 'OperationDefinition' && operation.operation === 'mutation'
 }
+
+async function validateApiVersion(adminSession: {token: string; storeFqdn: string}, version: string): Promise<void> {
+  if (version === 'unstable') return
+
+  const supportedVersions = await supportedApiVersions(adminSession)
+  if (supportedVersions.includes(version)) return
+
+  const formattedVersions = supportedVersions
+    .map((supportedVersion) => outputContent`${outputToken.green(supportedVersion)}`.value)
```

Comment:
I like the green color here, but it's not consistent with other commands, where we leave this kind of things with the default color. Maybe we can ask UX about it.

Reply:
@jenstanicak WDYT of the color use here? See screenshots in the PR description.

Reply:
ripped em out, shipping, TY for the feedback folks <3